Tests/RxSwift/nwiseTests.swift

```diff
@@ -12,6 +12,7 @@ import RxSwift
 import RxSwiftExt
 import RxTest
 
+// swiftlint:disable:next type_name
```

Comment:
I made this an exception since, even though you usually want Capitalized class named, in this case nwise would read NwiseTests, and It seemed odd. Any opinions welcomed :)

Reply:
`NWise` might look even more odd, but I'd prefer sticking to this rule in particular :)

Reply:
You mean `NwiseTests` or `nwiseTests` ?

Reply:
`NwiseTests`, but would suggest a second opinion as well :)